pkgs/top-level/python-aliases.nix

```diff
@@ -225,6 +225,7 @@ mapAliases ({
   Pyro4 = pyro4; # added 2023-02-19
   Pyro5 = pyro5; # added 2023-02-19
   PyRSS2Gen = pyrss2gen; # added 2023-02-19
+  pysha3 = safe-pysha3; # added 2023-05-19
```

Comment:
Probably just deprecate it instead, if additional postPatch is needed.

Reply:
Done